core/src/main/scala/com/softwaremill/sttp/SttpBackendOptions.scala

```diff
@@ -37,12 +40,38 @@ object SttpBackendOptions {
     }
   }
 
-  val Default: SttpBackendOptions = SttpBackendOptions(30.seconds, None)
+  private val Empty: SttpBackendOptions =
+    SttpBackendOptions(30.seconds, None)
+
+  val Default: SttpBackendOptions =
+    Empty.copy(proxy = loadSystemProxy)
 
   def connectionTimeout(ct: FiniteDuration): SttpBackendOptions =
-    Default.connectionTimeout(ct)
+    Empty.connectionTimeout(ct)
```

Comment:
I'm wondering, wouldn't the end-user expect `SttpBackendOptions.connectionTimeout` to change only the connection timeout comparing to the defaults? In other words, shouldn't we modify `Default` here?

Reply:
Wow. Did not think about it, but you are right. Fixed.